I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/locations.rpy", line 97, in script
    if entering:
  File "game/locations.rpy", line 97, in <module>
    if entering:
NameError: name 'entering' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/locations.rpy", line 97, in script
    if entering:
  File "renpy/ast.py", line 1898, in execute
    if renpy.python.py_eval(condition):
  File "renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/locations.rpy", line 97, in <module>
    if entering:
NameError: name 'entering' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Rogue-Like Chonky Mod 0.2i
Thu Jun 23 22:15:07 2022
